pages/dkp/kommander/2.2/licensing/essential/index.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: layout.pug
+navigationTitle: DKP Essential
+title: DKP Essential
+excerpt: A section containing all of the features available to users with a DKP Essential license.
+menuWeight: 5
+beta: false
+---
+
+## DKP Essential overview
+
+![DKP Essential Diagram](../img/dkpessentialdiagram.png)
+
+DKP Essential is a FIPS-compliant, self-managed single cluster Kubernetes solution that gives you a feature-rich, easy-to-deploy, and easy-to-manage entry-level cloud container platform. The DKP Essential license gives the user access to the entire Konvoy cluster environment, as well as the Kommander platform application manager for standing up crucial day-2 level production.
+
+## Compatible infrastructure
+
+DKP Essential operates across a range of cloud, on-premise, edge, and air-gapped infrastructures and has support for various OSs, including immutable OSs.
+
+Deployment options for DKP Essential include:
+
+* Bare metal servers
+* VMware vSphere   
+
+Cloud deployment options for DKP Essential include:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Google Cloud Platform (GCP)
+
+For the basics on standing up a DKP Essential cluster in one of the previously listed environments of your choice, see [Choose Infrastructure][choose-infrastructure].  
+
+<p class="message--note"><strong> NOTE:</strong> Infrastructure options are dependent upon license type. Some infrastructure deployment options listed within the corresponding link above may not be available for DKP Essential users. For an expanded list of supported infrastructure, see <a href="https://docs.d2iq.com/dkp/kommander/2.2/licensing/enterprise/">DKP Enterprise</a>.</p>
```

Comment:
I am not aware of any direct tie between licensing and what infra providers are supported.    

True, you cannot install Essential (or even Enterprise) on AKS/EKS, whereas Enterprise can provide lifecycle management for managed clusters based on those platforms, but I think this is a confusing way to word that.

@GoelDeepak  - is this really how we want to explain this?